### ISSUE TITLE
long descriptions for accessibility

### ISSUE THREAD

**Issue by philbarker** [K12OCX/k12ocx-specs#44]
visual content should have long descriptions. See, e.g., [WCAG longdesc](https://www.w3.org/TR/WCAG20-TECHS/H45.html) and mapping to NIMAS (issue #43)
